### DISCUSSION TITLE
Can you provide a case of a browser right-click pop-up menu?

### DISCUSSION THREAD

**Discussion by ys2rice2yangsong** [leptos-rs/leptos#2148] [Q&A]
Can you provide a case of a browser right-click pop-up menu?

**Comment by gbj** [comment]
(In the future, for a support question like this, rather than opening an issue and deleting the issue template, please open a question in discussions like this.)

I have never done this but a quick Google search told me that the `contextmenu` event is fired for this. You can use this like any other event. Here's a really simple example
```rust
#[component]
pub fn App() -> impl IntoView {
    let (show, set_show) = create_signal(false);

    window_event_listener(ev::contextmenu, move |ev| {
        ev.prevent_default();
        set_show.set(true);
    });

    view! {
        <Show when=show>
            <ol>
                <li>"Item 1"</li>
                <li>"Item 2"</li>
                <li>"Item 3"</li>
            </ol>
        </Show>
    }
}
```
